onnx_tf/opset_version.py

```diff
@@ -27,7 +27,7 @@
     'ConstantFill': [1],
     'ConstantOfShape': [9],
     'Conv': [1],
-    'ConvInteger': [],
+    'ConvInteger': [10],
```

Comment:
Please rebase and run gen_opset.py & gen_status.py to get the latest format and content for this file.